Giveaways.md

```diff
@@ -0,0 +1,15 @@
+# Giveaways
+From time to time DevStaff has the opportunity to offer some giveaways during the meetups. The prizes are provided by [sponsors](https://github.com/devstaff-crete/DevStaff-Heraklion/blob/master/Sponsors.md) in return of advertising.
+
+The process, the tools and the rules we use for these giveaways are described here:.
+
+# Prizes list
+All prizes will be written down at each meetup's announcement at [meetup.com](https://www.meetup.com/devstaff/). Moreover, at the intro presentation, when sponsors are presented, each prize will be explicitly mentioned as well as the type, the amount of prizes and the sponsor of each one.
```

Comment:
"as well as the type, ..." could be rephrased to "along with the type, ..."

Reply:
Changed at https://github.com/devstaff-crete/DevStaff-Heraklion/pull/462/commits/900ab9cf1b59d55703a6b1785a2b3dffd817f50b